aten/src/TH/vector/avx_mathfun.h

```diff
@@ -298,6 +299,89 @@ v8sf exp256_ps(v8sf x) {
   return y;
 }
 
+_PS256_CONST(tanh_0p625, 0.625f);
```

Comment:
I'm not so sure we want to put this here, `avx_mathfun.h` is like a library from http://software-lisc.fbk.eu/avx_mathfun/

Reply:
Since other functions in the file are also AVX2 versions of cephes library functions, I thought this will be a good place to add the `tanh256_ps` here. Please recommend which will be a good place to add this function.